packages/doenetml-worker/lib-doenetml-core/src/dast/ref_resolve.test.rs

```diff
@@ -73,7 +70,7 @@ fn can_resolve_names() {
         referent,
         Ok(RefResolution {
             node_idx: c_idx,
-            unresolved_path: Some(vec![PathPart {
+            unresolved_path: Some(vec![FlatPathPart {
```

Comment:
Do we still use `PathPart` at all? Is that how the data is serialized in? Or can we get rid of `PathPart`?

Reply:
`PathPart` is used with the `Dast`, so yes, the DoenetML is serialized in that form. When we convert to the `FlatDast`, it gets transformed into the `FlatPathPart`.